src/Adliance.AspNetCore.Buddy.Pdf/V2/AdliancePdfer.cs

```diff
@@ -26,14 +26,18 @@ public async Task<byte[]> HtmlToPdf(string html, PdfOptions options)
             {
                 Timeout = TimeSpan.FromMinutes(3)
             };
+
+            var paperSize = ConvertPaperSizeFromMMToPixels(options.Size, options.PaperWidth, options.PaperHeight);
             
             var parameters = new
             {
                 html,
                 footer = options.FooterHtml,
                 footer_height = options.FooterHeight,
                 header = options.HeaderHtml,
-                header_height = options.HeaderHeight
+                header_height = options.HeaderHeight,
+                paperWidth = paperSize[0],
```

Comment:
Can we please be consistent with the naming of the JSON variables -> the other variables use snake_casing, we should use 'paper_width' and 'paper_height'.